app/scripts/modules/core/src/search/externalSearch.registry.ts

```diff
@@ -38,7 +38,7 @@ export class ExternalSearchRegistry {
     this.registry[key] = searchConfig;
   }
 
-  public search(query: string | IQueryParams): IPromise<ISearchResult[]> {
```

Comment:
worth verifying this doesn't blow up with the search stuff we have in our internal build of deck